src/core/lib/event_engine/windows/windows_engine.h

```diff
@@ -97,8 +97,8 @@ class WindowsEventEngine : public EventEngine,
   // This is public because most classes that know the concrete
   // WindowsEventEngine type are effectively friends.
   // Not intended for external use.
-  ThreadPool* thread_pool() { return thread_pool_.get(); }
-  IOCP* poller() { return &iocp_; }
+  virtual ThreadPool* thread_pool() { return thread_pool_.get(); }
```

Comment:
Why is this needed? I see it overridden in the test, but it isn't used in the test.

Reply:
Removed.